cmake/util/Util.cmake

```diff
@@ -74,3 +74,31 @@ function(assign_source_group group)
         source_group("${group}\\${_source_path_msvc}" FILES "${_source}")
     endforeach()
 endfunction(assign_source_group)
+
+cmake_policy(SET CMP0057 NEW) # Needed for IN_LIST used in conditional
```

Comment:
this might come as a suprise to other cmake modules. would be great if we can avoid. e.g. we can write a few if after for instead since there are not a lot of cases

Reply:
see the regex example below